posthog/utils.py

```diff
@@ -313,6 +313,8 @@ def authenticate(self, request: Request):
 
 class PublicTokenAuthentication(authentication.BaseAuthentication):
     def authenticate(self, request: Request):
+        from django.contrib.auth.models import AnonymousUser
```

Comment:
I started getting `django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.` otherwise. 🤔 Would be happy to move it back to top level somehow though.

Reply:
Oh makes sense, we might be loading this module too soon